npm-scripts.mjs

```diff
@@ -360,40 +392,40 @@ function lintWorker()
 {
 	logInfo('lintWorker()');
 
-	executeCmd(`${MAKE} lint -C worker`);
+	executeCmd(`"${PYTHON}" -m invoke -r worker lint`);
 }
 
 function flatcNode()
 {
 	logInfo('flatcNode()');
 
 	// Build flatc if needed.
-	executeCmd(`${MAKE} -C worker flatc`);
+	executeCmd(`"${PYTHON}" -m invoke -r worker flatc`);
 
 	const buildType = process.env.MEDIASOUP_BUILDTYPE || 'Release';
 	const extension = IS_WINDOWS ? '.exe' : '';
 	const flatc = path.resolve(path.join(
 		'worker', 'out', buildType, 'build', 'subprojects', `flatbuffers-${FLATBUFFERS_VERSION}`, `flatc${extension}`));
-	const src = path.resolve(path.join('worker', 'fbs', '*.fbs'));
 	const out = path.resolve(path.join('node', 'src'));
-	const options = '--ts-no-import-ext --gen-object-api';
-	const command = `${flatc} --ts ${options} -o ${out} `;
 
-	if (IS_WINDOWS)
-	{
-		executeCmd(`for %f in (${src}) do ${command} %f`);
-	}
-	else
+	for (const dirent of fs.readdirSync(path.join('worker', 'fbs'), { withFileTypes: true }))
 	{
-		executeCmd(`for file in ${src}; do ${command} \$\{file\}; done`);
+		if (!dirent.isFile() || path.parse(dirent.name).ext !== '.fbs')
+		{
+			continue;
+		}
+
+		const filePath = path.resolve(path.join('worker', 'fbs', dirent.name));
+
+		executeCmd(`"${flatc}" --ts --ts-no-import-ext --gen-object-api -o "${out}" "${filePath}"`);
 	}
```

Comment:
@jmillan this change is needed to allow mediasoup installation in paths with spaces. See commit:

Allow mediasoup installation under path with whitespaces (fixes #753): https://github.com/versatica/mediasoup/pull/1239/commits/c7720f2da4f399a81d1b482cf28d8d604207ff19

And it makes logic simpler by running same command in Linux, macOS and Windows.